include/pacmod3/pacmod3_node.hpp

```diff
@@ -120,6 +120,7 @@ class PACMod3Node final
 
   VehicleType vehicle_type_;
   std::string frame_id_;
+  unsigned int dbc_major_version_;
```

Comment:
uint32_t is preferred.

Reply:
The rest of the code currently uses `unsigned int`. Should I break consistency and use uint32_t?

Reply:
It is fine for this one then. This is one note I had:
MISRA (directive 4.6): "typedefs that indicate size and signedness should be used in place of the basic types".

Reply:
Totally agree. I'll create a separate issue for it.